.github/workflows/release-please.yaml

```diff
@@ -38,6 +38,11 @@ jobs:
                 "type":"build",
                 "section":"Build System or External Dependencies",
                 "hidden":false
+              },
+              {
+                "type":"expr",
```

Comment:
Is "expr" the most intuitive short version of "experimental"? How about "exp"? https://www.abbreviations.com/abbreviation/experimental

Reply:
Sure, we can switch it to "exp" 👍